diff --git a/openni_tracker/src/openni_tracker.cpp b/openni_tracker/src/openni_tracker.cpp
index 3175ef0..ecd96b2 100644
--- a/openni_tracker/src/openni_tracker.cpp
+++ b/openni_tracker/src/openni_tracker.cpp
@@ -61,7 +61,7 @@ void publishTransform(XnUserID const& user, XnSkeletonJoint const& joint, string
     XnSkeletonJointPosition joint_position;
     g_UserGenerator.GetSkeletonCap().GetSkeletonJointPosition(user, joint, joint_position);
     double x = joint_position.position.X / 1000.0;
-    double y = joint_position.position.Y / 1000.0;
+    double y = -joint_position.position.Y / 1000.0;
     double z = joint_position.position.Z / 1000.0;
 
     XnSkeletonJointOrientation joint_orientation;
@@ -90,7 +90,7 @@ void publishTransforms() {
         if (!g_UserGenerator.GetSkeletonCap().IsTracking(user))
             continue;
 
-        string frame_id("openni_depth");
+        string frame_id("openni_depth_optical_frame");
 
         publishTransform(user, XN_SKEL_HEAD,           frame_id, "head");
         publishTransform(user, XN_SKEL_NECK,           frame_id, "neck");
